ui/src/router/components/form/validation/schema.js

```diff
@@ -84,6 +85,13 @@ const ruleConditionSchema = yup.object().shape({
 });
 
 const trafficRuleSchema = yup.object().shape({
+  name: yup
```

Comment:
Shall we also add the uniqueness check (which is easy to do on the frontend)? Similar to: https://github.com/caraml-dev/xp/blob/main/ui/src/experiments/components/form/validation/schema.js#L184